robot/EDA/resources/EDA.robot

```diff
@@ -230,72 +256,65 @@ Create HouseHold
+    Scroll Page To Location     50    400
     Click Related List Button   Organization Affiliations    New
-    Populate Lookup Field    Organization    &{account}[Name]
-    Click Modal Button    Save
-    [Return]         ${account_id}    ${contact_id}
+    Populate Lookup Field       Organization    &{account}[Name]
+    Click Modal Button          Save
+    [Return]                    ${account_id}    ${contact_id}
     
 Create Opportunities
-    [Arguments]    ${opp_name}    ${hh_name}  
+    [Arguments]                 ${opp_name}    ${hh_name}
     Select Window
-    Sleep    2   
+    Sleep                       2
     Populate Form
-    ...                       Opportunity Name= ${opp_name}
-    ...                       Amount=100 
-    Click Dropdown    Stage
-    Click Link    link=Closed Won
-    Populate Lookup Field    Account Name    ${hh_name}
-    Click Dropdown    Close Date
-    Pick Date    10
-    Select Modal Checkbox    Do Not Automatically Create Payment
-    Click Modal Button        Save
-    
-    
+    ...                         Opportunity Name= ${opp_name}
+    ...                         Amount=100
+    Click Dropdown              Stage
+    Click Link                  link=Closed Won
+    Populate Lookup Field       Account Name    ${hh_name}
+    Click Dropdown              Close Date
+    Pick Date                   10
+    Select Modal Checkbox       Do Not Automatically Create Payment
+    Click Modal Button          Save
+
 Choose Frame
-    [Arguments]    ${frame}
-    Select Frame    //iframe[contains(@title,'${frame}')]
+    [Arguments]                 ${frame}
+    Select Frame                //iframe[contains(@title,'${frame}')]
     
 Select Frame with ID
-    [Arguments]    ${id}
-    Select Frame    //iframe[contains(@id, '${id}')]    
+    [Arguments]                 ${id}
+    Select Frame                //iframe[contains(@id, '${id}')]
     
 Select Frame With Title
-    [Arguments]    ${name}
-    Select Frame    //iframe[@title= '${name}']    
+    [Arguments]                 ${name}
+    Select Frame                //iframe[@title= '${name}']
     
 Scroll Page To Location
-    [Arguments]    ${x_location}    ${y_location}
-    Execute JavaScript    window.scrollTo(${x_location},${y_location}) 
-
-Go To Eda Settings
-   Open App Launcher
-
-    # Check for EDA Tile
-    Wait Until Element Is visible           //div[@class='slds-app-launcher__tile-body']//a[contains(text(),'EDA')]
-
-    Wait Until Element Is visible           //a[@title='EDA Settings']//span[@class='label-ctr slds-app-launcher__tile-body slds-app-launcher__tile-body--small']//span[contains(text(), 'EDA Settings')]
-    Click Element                           //a[@title='EDA Settings']//span[@class='label-ctr slds-app-launcher__tile-body slds-app-launcher__tile-body--small']//span[contains(text(), 'EDA Settings')]
+    [Arguments]                 ${x_location}    ${y_location}
```

Comment:
I didn't add this line of code. It was existing as part of old tests. I just did a bit of formatting. I'd like to check with Doug before I make any changes, in case it breaks something.
Thanks for pointing it out (I'm no expert on this either), but I'll bring this up with him.